Dockerfile

```diff
@@ -62,7 +62,7 @@ RUN apt-get update; \
 ARG RUNNER_VERSION
 
 RUN pip install -U pip && pip install poetry && mkdir -p /root/.config/pypoetry \
-    && echo "[virtualenvs]" > /root/.config/pypoetry/config.toml \ 
+    && echo "[virtualenvs]" > /root/.config/pypoetry/config.toml \
```

Comment:
This change was automatically added by my IDE, let me know if it needs to be reverted